### PR TITLE
Tag Convex v0.5.0 [https://github.com/JuliaOpt/Convex.jl]

### PR DESCRIPTION
**Pull request by Ayush-iitkgp**
This is an improvement over the PR  #8080 
Please close the previous PR and merge this one.

### CODE REVIEW

File: Convex/versions/0.5.0/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.5-
```
Comment (tkelman) [inline Convex/versions/0.5.0/requires]: if you don't need to support prereleases, drop the trailing dash

Comment (Ayush-iitkgp) [inline Convex/versions/0.5.0/requires]: Not sure. We would want to keep support for prereleases. We are planning to completely drop support only after stable julia v0.6 release.

Comment (tkelman) [inline Convex/versions/0.5.0/requires]: prereleases of julia 0.5, which was released in september?

Comment (ViralBShah) [inline Convex/versions/0.5.0/requires]: Yes, the - should be dropped, since you don't want to support 0.5 pre-releases.

**Comment by Ayush-iitkgp** [comment]
All suggestions incorporated in the latest new PR by attobot. So sorry for the mess!

**Pull request closed without merging**